db/migrate/20150613202109_create_o_auth_applications.rb

```diff
@@ -0,0 +1,17 @@
+class CreateOAuthApplications < ActiveRecord::Migration
+  def self.up
+    create_table :o_auth_applications do |t|
+      t.belongs_to :user, index: true
+      t.string :client_id
+      t.string :client_secret
+      t.string :name
+      t.string :redirect_uris
+
+      t.timestamps null: false
+    end
+  end
+
+  def self.down
+    drop_table :o_auth_applications
+  end
```

Comment:
Yeah I think I did this to force a table drop after messing with the OAuthApplication model. I'll change it back to def change.